subworkflows/nf-core/fgbio_create_umi_consensus/main.nf

```diff
@@ -0,0 +1,86 @@
+//
+// Runs FGBIO tools to remove UMI tags from FASTQ reads
+// Convert them to unmapped BAM file, map them to the reference genome,
+// use the mapped information to group UMIs and generate consensus reads
+//
+
+params.groupumistrategy        = "Adjacency"
+params.aligner                 = "bwa-mem"
+
+include { FGBIO_FASTQTOBAM                  as FASTQTOBAM }         from '../../../modules/fgbio/fastqtobam/main'
+include { SAMTOOLS_BAM2FQ                   as BAM2FASTQ }          from '../../../modules/samtools/bam2fq/main.nf'
+include { BWA_INDEX }                                               from '../../../modules/bwa/index/main.nf'
+include { BWA_MEM }                                                 from '../../../modules/bwa/mem/main'
```

Comment:
I usually rename these as `BWAMEM1_INDEX` and `BWAMEM1_MEM` to have more cohesion with `BWAMEM2` modules.

Reply:
thanks for this. updated as suggested!